packages/govtnz-ds/build_src/react-ts-styled-components/InputBlock.tsx

```diff
@@ -271,21 +271,21 @@ const InputBlock = ({
         </StyledDiv3>
       </React.Fragment>
     ) : (
-      ""
+      ''
     )}
     <StyledInput
       width={width}
       error={error}
       aria-describedby={
         hintId !== undefined || errorId !== undefined
-          ? `${hintId ? hintId : ""}${errorId ? " " + errorId : ""}`
+          ? `${errorId ? ' ' + errorId : ''}${hintId ? hintId : ''}`
```

Comment:
I think this is the correct place @holloway  ?

Reply:
@Juliet-Brown These files (and any files in `govtnz-ds/build` or `govtnz-ds/build_src`) are generated by MetaTemplate so change the order in https://github.com/GOVTNZ/govtnz-design-system/tree/master/packages/govtnz-ds/src/template-sources and then `yarn build` to regenerate them.